### PR TITLE
GI Bill Comparison Tool content changes

### PR DESCRIPTION
**Pull request by rianfowler**
[Ticket](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/11471)

screen shot 2018-07-11 at 09 26 23
screen shot 2018-07-11 at 09 26 01
screen shot 2018-07-11 at 09 25 49
screen shot 2018-07-11 at 09 25 07

### CODE REVIEW

File: src/applications/gi/components/profile/CalculatorForm.jsx [hunk 1]
```diff
@@ -466,7 +466,7 @@ class CalculatorForm extends React.Component {
       <div>
         <RadioButtons
           label={this.renderLearnMoreLabel({
-            text: 'Will the majority of your classes be on campus?',
+            text: 'Will the majority of your classes be on main campus?',
```
Comment (cehsu) [inline src/applications/gi/components/profile/CalculatorForm.jsx]: This is the request change @briangryth, should we add a "the" before main?

File: src/applications/gi/containers/Modals.jsx [hunk 2]
```diff
@@ -329,7 +329,7 @@ export class Modals extends React.Component {
               target="_blank">this page.</a>
           </p>
           <p>
-            Only Veterans entitled to the maximum benefit rate or their designated transferees can receive this funding. Active duty Servicemembers and their spouses are not eligible for this program (child transferees of active duty Servicemembers may be eligible if the servicemember is qualified at the 100% rate). This information will be updated quarterly.
+            Veterans, Fry Scholarship, and Purple Heart recipients are entitled to the maximum benefit rate or their designated transferees can receive this funding. Active duty Servicemembers and their spouses are not eligible for this program (child transferees of active duty Servicemembers may be eligible if the servicemember is qualified at the 100% rate). This information will be updated quarterly.
```
Comment (cehsu) [inline src/applications/gi/containers/Modals.jsx]: This is the requested change from stakeholders, @peggygannon would it be of interest to reword this as "Veterans and Fry Scholarship and Purple Heart recipients" if that is clearer?

**Review by cehsu — APPROVED** [review]
LGTM. I had a few questions about the requested content for @briangryth or @peggygannon.

File: src/applications/gi/components/profile/CalculatorForm.jsx (hunk 1 above)
Comment (briangryth) [reply src/applications/gi/components/profile/CalculatorForm.jsx]: Yes that would be better

File: src/applications/gi/containers/Modals.jsx (hunk 2 above)
Comment (briangryth) [reply src/applications/gi/containers/Modals.jsx]: That works just fine
